app/geo/gui/src/ts/twxplore/gui/geo/attributeStrategies/attributeStrategies.ts

```diff
@@ -1,82 +1,156 @@
 // In an OOP Language -
+
+import { FeatureAttributeName } from "../states/map/FeatureAttributeName";
+
 // TypeScript
-interface FeatureAttributeFunctions {
+interface FeatureAttributeTypes {
```

Comment:
You don't need the interface AND the abstract class. Just keep the abstract class for now.